app/services/force/custom_api/lending_institutions_service.rb

```diff
@@ -5,32 +5,36 @@ module CustomApi
     # Provide Salesforce custom API interactions for rental assistances
     class LendingInstitutionsService < Force::Base
       def lending_institutions
+        self.class.lending_institutions(@client)
```

Comment:
i think that my future self would probably appreciate a comment somewhere in this file explicitly stating that the the values are being memoized and will be reloaded on server restart

Reply:
Sure, will add a comment